fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -249,6 +251,7 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         if(focusDrawerHandle) {
             persistentSheetBinding.sheetDrawerHandle.requestFocus()
         }
+        setImportantForAccessibility(backgroundViews!!, View.IMPORTANT_FOR_ACCESSIBILITY_YES)
```

Comment:
This might cause NPE. Can we add check instead

Reply:
Done